app/views/layouts/angular-bootstrap/_endpoints_angular.html.haml

```diff
@@ -77,6 +77,16 @@
                           "checkchange"             => "",
                           "prefix"                  => prefix.to_s,
                           "reset-validation-status" => "#{prefix}_auth_status"}
+      - detect = "detectClicked({target: '.detect_button'}, true)"
+    %span
+      %miq-button.detect_button{:ng_if => defined?(detection) ? true : false,
```

Comment:
We're doing `- foo ||= false` for the other variables on the beginning of this partial - can you do the same for `detection` please? (So that you don't need that `defined?`).

Also.. if you're trying to use ng-if, it has to be `'ng-if'`, not `:ng_if` .. and that `true` and `false` should probably be in quotes, unless you're interested in the presence of such attribute, instead of its value.